docs/TektonConfig.md

```diff
@@ -284,6 +289,39 @@ chain:
   transparency.url: #value
 ```
 
+### Result
+
+Result section allows user to customize the Tekton Result component, Refer to [Result Spec](https://github.com/tektoncd/operator/blob/main/docs/TektonResult.md#spec) section in TektonResult for available options.
+
+Example:
+
```

Comment:
Should we mention what's the default when no values are specified?

Reply:
@pratap0007 ^^^

Reply:
@khrm  updated and please review once again